app/waf/src/controllers/adc.controller.ts

```diff
@@ -346,4 +349,24 @@ export class AdcController {
       }
     });
   }
+
+  private async cUserdata(): Promise<string> {
+    let root_password: string = Math.random()
+      .toString(36)
+      .slice(-8);
+    let admin_password: string = Math.random()
+      .toString(36)
+      .slice(-8);
+
+    const user_data_unencoded: string = `#cloud-config
+    runcmd:
+     - "echo"root:{{ ${root_password} }}" | chpasswd"
```

Comment:
remove {{}}, that's postman's variable definition.

Reply:
roger